app/auth/templates/auth/forgot_password.html

```diff
@@ -13,11 +13,11 @@
     {{ form.csrf_token }}
     <div class="card-body p-6">
       <div class="card-title">Forgot password</div>
-      <p class="text-muted">Enter your email address and your will receive an email to reset your password.</p>
+      <p class="text-muted">Enter your email address you signed up with and you will receive an email to reset your password.</p>
 
       <div class="form-group">
         <label class="form-label">Email address</label>
-        {{ form.email(class="form-control", type="email", placeholder="Enter email") }}
+        {{ form.email(class="form-control", type="email", placeholder="Your email address associated with your SimpleLogin account") }}
```

Comment:
I'd put "The email address associated with your SimpleLogin account" as placeholder to avoid redundant "your"